modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase

```diff
@@ -440,6 +440,80 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12393. As a buyer I can view the UOMs sorted by priority."
```

Comment:
@description = "COMMERCE-12393. As a buyer I can view the UOMs sorted by priority on the UOM selector from the product details page